doctr/models/recognition/sar/pytorch.py

```diff
@@ -301,6 +308,7 @@ def _sar(
     backbone_fn: Callable[[bool], nn.Module],
     layer: str,
     pretrained_backbone: bool = True,
+    ignore_keys: Optional[List[str]] = None,
```

Comment:
Thats something to verify, but I think the sar_resnet31 pretrained checkpoint isn't working, and if that's the case, we can remove this 👍

Reply:
I disagree we have added this to make it possible to use the pretrained one to tune on new vocabs which will otherwise in case of a shape mismatch fail :)